Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.9-buster as base
+
+RUN pip install "poetry==1.1.4"
+COPY poetry.lock pyproject.toml .
+RUN poetry install
+
+COPY ./todo_app /todo_app/
+
+From base as production
```

Comment:
We should add some instructions to the Readme for how to use these images; it's easy enough for us to know how to run `docker compose up` but even building/running your production image requires some knowledge of e.g. what ports to expose - which will very quickly slip out of your mind if you don't look at this for a while! - so it would be good to record the relevant build/run commands